### ISSUE TITLE
Potentially copied commercial product on Steam

### ISSUE THREAD

**Issue by coornio** [SebLague/Geographical-Adventures#121]
I found this, and I feel like your license prohibits such a project if you're not the one who initiated publishing. If not, it seems some studio decided to rip your project for a quick buck:

https://store.steampowered.com/app/1830130/Planetary_Delivery/

**Comment by JadoTech** [comment]
I bought it out of curiosity and its terrible, but you're right its too similar not to be copied. But the dev has obviously changed a bunch of things to attempt to make it different. Refunded of course!

**Comment by GeorgeS2019** [comment]
@coornio 
Thx for sharing. 
The codes here and the video associated are much more inspiring.

Any attempt to make a commercial version has to do a MUCH better job to avoid others overtaking their commercial effort.

**Comment by coornio** [comment]
> Any attempt to make a commercial version has to do a MUCH better job to avoid others overtaking their commercial effort.

Sure it would, but I am also pretty sure that said commercial project would also require permission from the author of the code and assets to utilize said resources. This is a blatant ripoff with minor asset swaps, and I wasn't able to find any note anywhere regarding that.

**Comment by adamency** [comment]
@SebLague sorry to ping you, but I feel you would want to be aware of this rather now than later

Thanks a lot @coornio for the alert :)

**Comment by DreadKyller** [comment]
The license of the project according to Github was set to MIT license, which does allow for modifying and selling the content, so while a bit rude of the person who created the sold version to not credit where they got their base game from (which is pretty clear) there's unfortunately nothing wrong from a legal perspective it seems unless the MIT license header isn't included with the game. The only question would be whether it's against the license of any of the assets used in this project.

**Comment by adamency** [comment]
@DreadKyller Not at all, you're wrong on the timeline by a long shot. The license was changed to MIT on 2024-02-01 while the paid game in question was release on 2023-06-20, i.e. more than 7 months before this change. And at that time, the license was Creative Commons which IIRC does NOT allow commercial use of any of the content. If that is right, this is legally actionable against the author of the steam game.
